WordPress/Classes/Models/Blog.m

```diff
@@ -27,6 +28,7 @@
 @interface Blog ()
 
 @property (nonatomic, strong, readwrite) WPXMLRPCClient *api;
+@property (nonatomic, strong, readwrite) WordPressOrgXMLRPCApi *xmlRpcApi;
```

Comment:
I think for consistency this should be `xmlrpcApi`. We normally don't capitalize the `r` and it looks kind of funny.